content/en/xlt/load-testing/Advanced/210-gradle-builds.md

```diff
@@ -0,0 +1,92 @@
+---
+title: "Gradle"
+
+weight: 210
+type: docs
+
+description: >
+    How to build and run using Gradle.
+---
+
+XLT supports using Gradle as a build tool for your test suite. You need the following entries in your `build.gradle` file:
+
+## Integrating XLT into a Gradle project
+
+### Plugins
+Include the following in your `build.gradle` file to use Gradle's Java plugin and its provided build steps:
+
+```groovy
+plugins {
+    id 'java'
+}
+```
+
+### Adding Repository Location
+XLT is published to [Maven Central](https://search.maven.org/artifact/com.xceptance/xlt). To integrate XLT into your Gradle project, copy and paste the following into your `build.gradle` file:
+
+```groovy
+/* Add Maven Central repository information */
+
+repositories {
+    maven {
+        url = uri('https://repo.maven.apache.org/maven2/')
+    }
+}
+```
+
+### Dependency Scopes
+XLT is provided at runtime by the container, so it does not need to be packaged with your project build, thus reducing the upload size when starting a load test. Maven has the `provided` scope for dependencies that are needed in your classpath at compile time but not at runtime; however since there is no such scope defined in Gradle you have to define it yourself in your `build.gradle` file: 
+
+```groovy
+configurations {
+    provided
+}
+
+sourceSets {
+    main {
+         // Add dependencies of type 'provided' to compile classpath
+        compileClasspath += configurations.provided
+    }
+}
+```  
+
```

Comment:
In case your testsuite's code is organized in a different way, for example when your XLT tests resides in `src/test/java`, you may have to update the compile classpath of the proper [source set](https://docs.gradle.org/7.4.2/userguide/java_plugin.html#source_sets) as well.